frontend/integration-tests/tests/olm/prometheus.scenario.ts

```diff
@@ -1,23 +1,22 @@
 /* eslint-disable no-undef, no-unused-vars */
```

Comment:
The Prometheus Operator is better for e2e testing because it supports `SingleNamespace` install mode. The etcd Operator runs cluster-wide.